### ISSUE TITLE
Need help for using my datas in real-time

### ISSUE THREAD

**Issue by Wakarox** [lucastabelini/LaneATT#124]
Hello,
I modified the code to use it in real time in an ROS environment.
I need to know where do you modify your inputs in the model.
I have actually a '720, 1280, 3' image from cv2.
Your model need '64, 3, 7, 7'.
Can you have some times to explain me ?

**Comment by Wakarox** [comment]
@lucastabelini

**Comment by ilaij0810** [comment]
have you resized to 360*640 ?

**Comment by Wakarox** [comment]
I tried, but i don't know what do do next.

**Comment by lucastabelini** [comment]
The model does not need a 64x3x7x7 input. The input is an image resized do 512x288, as described in the paper.

**Comment by Wakarox** [comment]
Yes, but you use the Dataloader to import it. And the dataloader changed all the dimensions of the input using batch size, etc.
I would try to delete the Dataloader, is it possible ?

**Comment by lucastabelini** [comment]
No.
